android/src/main/java/com/thebluealliance/androidclient/models/Match.java

```diff
@@ -356,7 +357,6 @@ public MatchListElement render(boolean showVideo, boolean showHeaders, boolean s
         try {
             videos = getVideos();
         } catch (FieldNotDefinedException e) {
-            Log.w(Constants.LOG_TAG, "Required field for match render: Database.Matches.VIDEOS. Defaulting to none.");
```

Comment:
I think this is partially slowing down the renders (lots of logging is slow) and causing frame drops, since this is done all on the UI thread.

I can put it back and make it log.d so it doesn't affect prod though